project/AkkaBuild.scala

```diff
@@ -136,9 +136,9 @@ object AkkaBuild extends Build {
     id = "akka-cluster-sharding",
     base = file("akka-cluster-sharding"),
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm",
-        persistence % "compile;test->provided", clusterTools)
+        persistence % "compile;test->provided", distributedData % "compile;test->provided", clusterTools)
```

Comment:
Currently it is the "first step" and I removed the persistence just out of the coordinator. But the persistence stay alive inside the PersistenceShard. This is needed when `remember-entities = on`.